### PR TITLE
Update to 0.8.0

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Update to 0.8.0. Needed for libmambapy-1.5.3.

conda-forge: https://github.com/conda-forge/yaml-cpp-feedstock/blob/main/recipe/meta.yaml

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,22 @@
-{% set version = "0.7.0" %}
-{% set hash = "43e6a9fcb146ad871515f0d0873947e5d497a1c9c60c58cb102a97b47208b7c3" %}
-{% set build_number = "1" %}
+{% set version = "0.8.0" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:2]:
```suggestion
{% set name = "yaml-cpp" %}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
 
 package:
   name: yaml-cpp-split
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:4]:
```suggestion
  name: {{ name }}-split
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,30 +1,22 @@
...
-  sha256: {{ hash }}
-  patches:
-    # These patches fix a shared object issue in `0.7.0`. This is
-    # currently fixed in `master`, so future updates should not need
-    # them.
-    - include_yaml-cpp_dll.h.patch               # [win]
-    - include_yaml-cpp_node_detail_node.h.patch  # [win]
-    # Fix a defect in the cmake config file.
-    - yaml-cpp-config.cmake.in.patch
+  url: https://github.com/jbeder/yaml-cpp/archive/refs/tags/{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:8]:
```suggestion
  url: https://github.com/jbeder/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,30 +1,22 @@
...
+  number: 0
 
 requirements:
   build:
-    - patch     # [unix]
-    - m2-patch  # [win]
+    - patch  # [linux and (aarch64 or ppc64le or s390x)]
 
 outputs:
   - name: yaml-cpp
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]:
```suggestion
  - name: {{ name }}
```

Comment (skupr-anaconda) [reply recipe/meta.yaml:22]: and {{ name }}-static

Comment (skupr-anaconda) [reply recipe/meta.yaml:22]: I would use `ninja-base` instead of `ninja` for all outputs

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:2]: As is usual with me, I would like to not diverse too much from conda-forge if possible.

**Review by ryanskeith — APPROVED** [review]
LGTM.

**Review by cbouss — COMMENTED** [review]
Could we update the feedstock to match conda-forge? Especially, `yaml-cpp-static` depending on `yaml-cpp` does not seem right to me.

**Comment by JeanChristopheMorinPerso** [comment]
Indeed @cbouss . Though, can I request to defer this in a second PR so that I can unblock the mamba feedstock?

**Review by cbouss — APPROVED** [review]
Improvements put back to later

**Pull request merged**